test/test_conversion_driver_esm1p5.py

```diff
@@ -142,3 +152,32 @@ def test_convert_esm1p5_output_dir_error():
         esm1p5_convert.convert_esm1p5_output_dir(
             "/test_convert_esm1p5_output_dir_error/fake/path/"
         )
+
+def test_report_results_success():
+    succeeded_files = ["fake_file_1", "fake_file_2", "fake_file_3"]
+    # succeeded argument to report_results typically a list of Path objects
+    succeeded = [Path(p) for p in succeeded_files] 
+    failed = []
+
+    with mock.patch("builtins.print", return_value = None) as mock_print:
```

Comment:
I've added a comment further down, for modifying the code to avoid `print` mocking.